openpype/cli.py

```diff
@@ -131,6 +152,25 @@ def publish(debug, paths, targets):
     PypeCommands.publish(list(paths), targets)
 
 
+@main.command()
+@click.argument("path")
+@click.option("-d", "--debug", is_flag=True, help="Print debug messages")
+@click.option("-h", "--host", help="Host")
+@click.option("-u", "--user", help="User email address")
+@click.option("-p", "--project", help="Project")
+@click.option("-t", "--targets", help="Targets module", default=None,
```

Comment:
`targets` are pyblish targets not module targets. Pyblish target is for more granular filtering (e.g. "farm") has nothing to do with modules by design.